GitUI/CommandsDialogs/RsaUtil.cs

```diff
@@ -0,0 +1,365 @@
+﻿using System;
+using System.IO;
+using System.Numerics;
+using System.Security.Cryptography;
+using System.Text;
+using System.Windows.Forms;
+
+using JetBrains.Annotations;
+
+namespace GitUI.CommandsDialogs
+{
+	/// <summary>
+	/// RSA key formats manip.
+	/// </summary>
+	public static class RsaUtil
```

Comment:
Move OK.
Don't know what kind of tests would do here. Ideally, should test if openssh eats them, but that's no simple thing to do. Would feel safer with tests of course. Comparing to a predefined string might help, but not much.